src/kt_class.cpp

```diff
@@ -24,3 +33,17 @@ StringName KtClass::get_super_class(jni::Env& env) {
     jni::JObject ret = wrapped.call_object_method(env, getter);
     return StringName(env.from_jstring(jni::JString((jstring) ret.obj)));
 }
+
+KtMethod* KtClass::get_method(const StringName& methodName) {
```

Comment:
I prefer return a reference here, `KtClass` still owns the returned `KtMethod` - passing a pointer gives the caller an option to call delete.